src/Build.UnitTests/BuildEventArgsSerialization_Tests.cs

```diff
@@ -160,11 +160,15 @@ public void RoundtripTaskStartedEventArgs()
                 projectFile: "C:\\project.proj",
                 taskFile: "C:\\common.targets",
                 taskName: "Csc");
+            args.LineNumber = 42;
+            args.ColumnNumber = 999;
```

Comment:
smallest nit ever:
I have trouble imagining a file that looks like this...999+ lines happens fairly often, but 999 columns sounds like bad style.